src/compiler/transformers/es2015.ts

```diff
@@ -3173,6 +3173,7 @@ namespace ts {
         function visitCatchClause(node: CatchClause): CatchClause {
             const ancestorFacts = enterSubtree(HierarchyFacts.BlockScopeExcludes, HierarchyFacts.BlockScopeIncludes);
             let updated: CatchClause;
+            Debug.assert(!!node.variableDeclaration, "Catch clauses should always be present when downleveling ES2015 code.");

```

Comment:
How about `"Catch clause variable should always be present when downleveling ES2015"`?